IPython/utils/traitlets.py

```diff
@@ -1635,14 +1635,14 @@ def validate_elements(self, obj, value):
 class Dict(Instance):
     """An instance of a Python dict."""
 
-    def __init__(self, default_value=None, allow_none=True, **metadata):
+    def __init__(self, default_value={}, allow_none=True, **metadata):
```

Comment:
I think the actual default value will end up being `dict({})`, which should create a new copy for each, see this line of code: https://github.com/ipython/ipython/blob/master/IPython/utils/traitlets.py#L891

But I'm glad you prompted me to go and check that.